app/enterprise/0.34-x/getting-started/quickstart.md

```diff
@@ -18,6 +18,22 @@ run without containers by following our [CentOS][centos] or
 
 ## 1. Start Kong Enterprise
 
+If you will be securing your Admin API with RBAC or your Kong Manager 
+application with an authentication plugin, you will need a Super Admin 
+account, which is created during database migrations. 
```

Comment:
See style guide for personal pronouns. Avoid the use of "you" in favor of specific roles or the imperative mood.

"A Super Admin account is necessary in order to secure the Admin API with RBAC or the Kong Manager application with an authentication plugin. The Super Admin account is created during database migrations."